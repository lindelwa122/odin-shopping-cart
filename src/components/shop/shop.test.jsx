import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Shop from './shop';

describe('Shop Component', () => {
  it('renders correctly', () => {
    const { container } = render(<Shop />);
    expect(container).toMatchSnapshot();
  });
});
