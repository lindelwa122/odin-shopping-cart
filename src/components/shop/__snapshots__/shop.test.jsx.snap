// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Product Component > renders correctly 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;

exports[`S Component > renders correctly 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;

exports[`Shop Component > renders correctly 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;
